.github/workflows/lint-test.yaml

```diff
@@ -53,3 +53,15 @@ jobs:
 
       - name: Run chart-testing (install)
         run: ct install --target-branch main
+
+      - name: Run KUTTL smoke tests
+        run: |
+          sleep 10s
```

Comment:
Why sleep needed?

Reply:
It's because chart-testing deletion needs some time to finish, which we need to wait until the namespace is deleted. Otherwise, we will get an error.

Reply:
Which chart-testing deletion do you mean? I don't see any deletes prior to this

Reply:
This one: `ct install --target-branch main` in line 55. It will delete the chart after the testing.

Reply:
Can we use `kubectl wait --for=delete` instead?

Reply:
I think we can. I will make the changes.

Reply:
I tried to use `kubectl wait` command to wait until the ns is deleted. But it sometimes won't work and will cause time out error because the ns could be deleted before this command runs. @dmitryax Please let me know if I miss something. Thanks!

Reply:
I guess you can do something like this 
```
until kubectl get ns opentelemetry-operator-system 2>&1 >/dev/null; do sleep 1; done
```
It'll be more descriptive them just sleep for 10 seconds

Reply:
Using `until` command makes more sense. I have made the changes. Thanks.